### PR TITLE
add carbon emit API

### PR DESCRIPTION
**Pull request by guotuofeng**
add carbon emit API to show the emitted co2 per hour

MicrosoftTeams-image

### CODE REVIEW

**Review by guyang3532 — APPROVED** [review]

File: tb_plugin/torch_tb_profiler/consts.py [hunk 1]
```diff
@@ -67,3 +67,35 @@
...
+    "Titan RTX": 280,
+    "Titan V": 250,
+    "TITAN X Pascal": 250,
+    "Titan Xp": 250,
+    "TPUv2 Chip": 221,
+    "TPUv3 Chip": 283
+}
+
+DEFAULT_IMPACT = 0.432 # kg/kwH
+DEFAULT_OFFSET = 0
```
Comment (gaoteng-git) [inline tb_plugin/torch_tb_profiler/consts.py:101]: Add a newline?

File: tb_plugin/torch_tb_profiler/consts.py [hunk 2]
```diff
@@ -67,3 +67,35 @@
...
+    "Tesla V100-SXM2-32GB": 300,
+    "Titan RTX": 280,
+    "Titan V": 250,
+    "TITAN X Pascal": 250,
+    "Titan Xp": 250,
+    "TPUv2 Chip": 221,
+    "TPUv3 Chip": 283
+}
+
+DEFAULT_IMPACT = 0.432 # kg/kwH
```
Comment (gaoteng-git) [inline tb_plugin/torch_tb_profiler/consts.py:100]: Add a comment saying 'Use value of "Private Infrastructure" Provider which uses OECD's 2014 yearly average'?

File: tb_plugin/torch_tb_profiler/utils.py [hunk 3]
```diff
@@ -24,3 +24,11 @@ def get_logger():
 
 def is_chrome_trace_file(path):
     return consts.WORKER_PATTERN.match(path)
+
+def get_carbon_emit(gpu):
+    watt = consts.WATT_MAP.get(gpu)
```
Comment (gaoteng-git) [inline tb_plugin/torch_tb_profiler/utils.py:29]: Not sure whether the value of list(profile.gpu_infos.values())[0]['Name'] will exactly match keys in consts.WATT_MAP.
Where do you get the keys of consts.WATT_MAP?

**Review by gaoteng-git — COMMENTED** [review]
Cool!

**Pull request closed without merging**